tabu.sty

```diff
@@ -1298,6 +1306,61 @@
 \def\tabu@endofcollect #1{\ifnum0=`{}\fi
                           \expandafter\endgroup \the\toks@  #1%
 }% \tabu@endofcollect
+%% TABU-FIXED (issue #3)
+%% collects body with a catcode 12 backslash for later scantokenization
+\catcode`\| \z@
+\catcode`\\ 12
+|long|def|tabu@collectbody@forscan #1#2#3\end{%
+    |def|tabu@collectbody@forscan@next{|tabu@collectbody@forscan{#1}}%
+    |def|tabu@collectbody@forscan@partial{#2#3}%
```

Comment:
This causes

```
! Illegal parameter number in definition of \tabu@collectbody@forscan@partial
```

if body contains a `#` (e.g. \verb|#|, for example as in test file of issue #6).